lib/compare_all.js

```diff
@@ -38,9 +38,15 @@ var compareAll = function(newVersion, oldVersion, callback) {
       callback(err);
     }
 
-    callback(err, results.reduce(function(problems, result) {
-      return extend(problems, result);
-    }, {}));
+    callback(
+      err,
+      results.reduce(
+        function(problems, result) {
+          return extend(problems, result);
+        },
+        {}
+      )
+    );
```

Comment:
@bkowshik the whole philosophy is a bit opinionated. It might not please every eye. But on the brighter side, it keeps things consistent / predictable. The author talks more about the philosophy [here](http://jlongster.com/A-Prettier-Formatter?utm_content=bufferc7879&utm_medium=social&utm_source=facebook.com&utm_campaign=buffer).